### PR TITLE
Show dispatch message for 5 minutes

### PR DESCRIPTION
**Pull request by CKegel**

### CODE REVIEW

File: public/js/whiteboard.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const latitude = 42.72927;
 //Time Constants(in Milliseconds)
 const oneMinute = 60000;
 const thirtyMinutes = oneMinute * 30;
-const threeMinutes = oneMinute * 3;
+const threeMinutes = oneMinute * 5;
```
Comment (ddbruce) [inline public/js/whiteboard.js:9]: Except `threeMinutes` shouldn't equal 5

Maybe do `timeoutTime` and make that configurable.

**Review by ddbruce — CHANGES_REQUESTED** [review]

File: public/js/whiteboard.js (hunk 1 above)
Comment (CKegel) [reply public/js/whiteboard.js:9]: I was hoping you wouldn't notice...

Comment (CKegel) [reply public/js/whiteboard.js]: Should I make it configurable in the environment variable, or is what I've done acceptable?

**Review by ddbruce — APPROVED** [review]
lgtm

**Pull request merged**